tests/test_accessibility.py

```diff
@@ -29,11 +29,15 @@ class AccessibilityTest(WebAppTest):
     @patch.dict(os.environ, {'SELENIUM_BROWSER': 'phantomjs'})
     def setUp(self):
         super(AccessibilityTest, self).setUp()
+
+        # Name is badAriaAttributeValue
```

Comment:
oh sorry you had to trace this back! maybe for the next person, in the comment be explicit: "Name" -> "Audit rule name"